Containerfile

```
@@ -48,6 +48,8 @@ RUN microdnf -y --setopt=install_weak_deps=0 --setopt=tsflags=nodocs install \
     cyrus-sasl-lib cyrus-sasl-plain cyrus-sasl-gssapi openssl \
     python3 \
     libnghttp2 \
+    curl \
+    nmap-ncat \
```

maybe put these on the same line? bash does not really allow to put comments between these lines, so the grouping is the only way to provide an idea what the deps are for. and since both are test dependencies, i"d put them on the same line
```suggestion
    curl nmap-ncat \
```

---

Will do. I noticed that even after including nmap-ncat, the ncat tcp adaptor tests don't run. 

On an unrelated note, I am worried that including the packages needed for testing will increase the likelihood that the resulting skupper-router image might be involved in unneeded CVEs that might arise from these packages.

---

> On an related note, I am worried that including the packages needed for testing will increase the likelihood that the resulting skupper-router image might be involved in unneeded CVEs that might arise from these packages.

It won't, we use https://docs.docker.com/build/building/multi-stage/ in our containerfile